### DISCUSSION TITLE
about light intensity

### DISCUSSION THREAD

**Discussion by JackKaiXing** [google/filament#4369] [Q&A] [answered]
Hello, 

What is the recommended point light `intensity` for Tungsten lamp?

Here in filament file [LightManager.cpp](https://github.com/google/filament/blob/main/filament/src/components/LightManager.cpp), the default is `float mIntensity = 100000.0f;`. For point light, the SI UNIT is Candela, but I think 100000 is too much bright. However, in my test using filament, 100000*100 seem to be an OK intensity.

Any ideas?

**Comment by romainguy** [comment, marked as answer]
There is no recommended intensity, it depends on the type of light you are trying to replicate. Note that the intensity is affected by the camera exposure. The default camera exposure we use is meant for a bright sunny day outdoors, which is why you have to use a very high intensity for this light. Change your camera exposure.
